.github/workflows/native-wsl.yml

```diff
@@ -0,0 +1,175 @@
+name: Native and WSL
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    defaults:
+        run:
+          shell: ${{ matrix.configuration == 'wsl' && 'wsl-bash {0}' || 'pwsh' }}
+    strategy:
+      matrix:
+        os: [windows-latest]
+        node-version: [16, 14, 12, 10]
+        configuration: [wsl, native]
+        architecture: [x86, x64]
+    steps:
+    - uses: actions/checkout@v4
+    - uses: Vampire/setup-wsl@v3
+      if: matrix.configuration == 'wsl'
+      with:
+        distribution: Ubuntu-22.04
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v4
+      if: matrix.configuration == 'native'
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Do everything here because npm is not found when running in separate steps
+      if: matrix.configuration == 'wsl'
+      run: |
+        export ESLINT_VERSION=7
+        sudo apt update -y
+        sudo apt-get install curl -y
+        curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/master/install.sh | bash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+        [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+        nvm install ${{ matrix.node-version }}
+        nvm use ${{ matrix.node-version }}
+        node --version
+        if [ ${{ matrix.node-version }} -lt 11 ]; then
+          npm install --legacy-peer-deps
+        fi
+        if [ ${{ matrix.node-version }} -gt 10 ]; then
+          npm install
+        fi
+        npm install nyc@latest --no-save
+        set TRAVIS_NODE_VERSION=${{ matrix.node-version }}
+        if [ ${{ matrix.node-version }} -le 8 ]; then
+          set ESLINT_VERSION=6
+        fi
+        if [ ${{ matrix.node-version }} -gt 8 ]; then
+          set ESLINT_VERSION=7
+        fi
+        if [ ${{ matrix.node-version }} -ge 11 ]; then
+          set ESLINT_VERSION=8
+        fi
+        npm run copy-metafiles
+        # bash ./tests/dep-time-travel.sh 2>&1
+        npm install eslint@${ESLINT_VERSION} --no-save --ignore-scripts
+        if [ ${{ matrix.node-version }} -lt 12 ]; then
+          npm install @typescript-eslint/parser@4
+        fi
+        if [ ESLINT_VERSION -eq 8 ]; then
+          npm i --no-save eslint-plugin-import@'.' -f
+          npm run build
+        fi
+
+        git config core.symlinks true
+        git reset --hard
+        npm config set package-lock false
```

Comment:
this should already be set via .npmrc

Reply:
I don't see it in the root level `.npmrc` file.

Reply:
https://github.com/import-js/eslint-plugin-import/blob/c77c1a6ace311f99ed25878768c2dbee49c0512a/.npmrc#L1